central/src/gitlab.rs

```diff
@@ -24,6 +24,28 @@ pub struct GitLabProjectAccessTokenProvider {
     client: reqwest::Client,
 }
 
+/// Derive the bridgehead configuration repository from the beam id of the requester.
+/// Example return value: "bridgehead-configurations/bridgehead-config-berlin"
+fn derive_bridgehead_config_repo_from_beam_id(requester: &AppId) -> Result<String, String> {
+    let mut parts = requester.as_ref().splitn(3, '.');
+    let (_app, proxy, broker) = (parts.next().unwrap(), parts.next().unwrap(), parts.next().unwrap());
+
+    let group = match broker {
+        "broker.ccp-it.dktk.dkfz.de" => "bridgehead-configurations", // ccp
+        "broker.bbmri.sample.de" => "bbmri-bridgehead-configs", // bbmri
+        "broker.bbmri.de" => "bbmri-bridgehead-configs", // gbn
+        "test-no-real-data.broker.samply.de" => "bridgehead-configurations", // cce, itcc, kr
+        "broker.hector.dkfz.de" => "dhki", // dhki
+        _ => return Err(format!("Bridgehead configuration repository group not known for broker {broker}")),
+    };
+
+    Ok(match group {
+        "bridgehead-configurations" => format!("{group}/bridgehead-config-{proxy}"),
+        "dhki" => format!("{group}/{proxy}-bk"),
+        _ => format!("{group}/{proxy}"),
+    })
```

Comment:
Future work: Would it be possible to externalize this to e.g. a config file?

Reply:
To get rid of the first `match` we could use Jans idea and make it a config option for the secret sync central component. If I understand correctly, each broker has their own secret sync central component so that should work fine.

For the second `match` I could add a config option where you can specify an optional prefix for the repository name.